cmd/lock-rpc-server.go

```diff
@@ -20,7 +20,7 @@ import (
 	"fmt"
 	"math/rand"
 	"net/rpc"
-	"path"
+	pathutil "path"
```

Comment:
the problem is that `path` variable generally just conflicts with the package name and sadly Go doesn't complain properly .. it is just better as a safety. I can keep it `path` as well i don't forse usage if `path` as a variable anytime in future.